nextjs-javascript/next.config.js

```diff
@@ -1 +1,4 @@
-module.exports = {};
+module.exports = {
+	webpack5: false,
```

Comment:
maybe we can leave a note here why we can't use webpack5? it seems that many people complaint about that https://github.com/tradingview/charting-library-examples/issues/224